autoinstrumentation/java/Dockerfile

```diff
@@ -4,10 +4,43 @@
 #  - Grant the necessary access to the jar. `chmod -R go+r /javaagent.jar`
 #  - For auto-instrumentation by container injection, the Linux command cp is
 #    used and must be availabe in the image.
-FROM busybox
+
+# Stage 1: Build the cp-utility binary
+FROM rust:1.69 as builder
```

Comment:
can the rust cp impl. be built for all target platforms supported by go? What I have in mind are ibm z + power (that's what we have to support).

Reply:
yes, s390x and ppc64le are supported by the rust base image and musl.

Are you using the autoinstrumentation images that are published from this repo in those platforms? I'm asking because today they only support arm and x86 (please take a look [here](https://github.com/open-telemetry/opentelemetry-operator/blob/main/.github/workflows/publish-autoinstrumentation-java.yaml#L69))